processor/tailsamplingprocessor/README.md

```diff
@@ -18,6 +18,11 @@ Multiple policies exist today and it is straight forward to add more. These incl
 - `status_code`: Sample based upon the status code (`OK`, `ERROR` or `UNSET`)
 - `string_attribute`: Sample based on string attributes value matches, both exact and regex value matches are supported
 - `rate_limiting`: Sample based on rate
+- `composite`: Sample based on a combination of above samplers, with ordering and rate allocation per sampler. Rate allocation allocates certain percentages of spans per policy order. 
+  For example if we have set max_total_spans_per_second as 100 then we can set rate_allocation as follows
+  1. test-composite-policy-1 = 50 % of max_total_spans_per_second = 50 spans_per_second
+  2. test-composite-policy-1 = 25 % of max_total_spans_per_second = 25 spans_per_second
+  3. To ensure remaining capacity is filled use always_sample as one of the policies
```

Comment:
I couldn't find the code for this behavior. It would be worth adding tests asserting that this is the case too.

Reply:
Sure, Will do